test/websocket-reconnect.js

```diff
@@ -0,0 +1,40 @@
+import WebSocket from 'ws'
+import test from 'ava'
+import openWebSocket from '../src/open-websocket'
+
+const TEST_SERVER_PORT = 8080
+
+test.beforeEach(t => {
```

Comment:
Won't `beforeEach` run this hook before every other test that ava will run that have no link to the websocket reconnection?

Reply:
Yes. I wanted to use `test.before` to avoid that but then we need to upgrade to the latest version of ava because in v0.22 (which we use) there is no `context` available in `test.before`. I filed a ticket for it: https://github.com/avajs/ava/issues/1736

Reply:
I see, maybe simply create a `let` in that file then in the meantime

Reply:
Good idea! Done. ✨